packages/gatsby-theme-carbon/src/components/Video/Video.module.scss

```
@@ -5,7 +5,7 @@
 }
 
 .video-container {
-  margin-bottom: $spacing-08;
+  margin-bottom: $carbon--layout-05;
```

A few months ago we added all the layout values to the spacing scale so we wouldn't have to alternate between the two. You also don't need the carbon prefix.

```suggestion
  margin-bottom: $spacing-10;
```